include/dlaf/util_cublas.h

```diff
@@ -48,6 +48,21 @@ inline constexpr cublasFillMode_t blasToCublas(const blas::Uplo uplo) {
   }
 }
 
+/// Predicate returning true for all coordinates lying in the lower part or on the diagonal of a matrix
+__device__ inline constexpr bool isLower(unsigned i, unsigned j) noexcept {
+  return i >= j;
+}
```

Comment:
As mentioned already to @albestro these could just be `inline constexpr` with the `--expt-relaxed-constexpr` flag to `nvcc`. Basically it allows using constexpr functions anywhere. But this is also fine since the CUDA headers define `__device__` to nothing for non-CUDA compilers.